frontend/pages/404.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Link from "next/link";
+import { Button } from "react-bootstrap";
+import classes from "../styles/error.module.css";
+
+const hello = () => {
+  return (
+  <>
+        <img
+          className={classes.image}
+          src="/images/errorImg.png"
+          alt="timer"
+          width={"100vh"}
+          height={"100vh"}
+        />
+
+      <div className={classes.button}>
+      <Link href="/">
```

Comment:
instead of button you can add a text `404 page not found`

Reply:
![Screenshot (197)](https://github.com/J0SAL/Decentralized-Expense-Tracker/assets/98577878/0da9ddbf-9328-4002-b3d2-39e0cf9fe0bd)
Any other changes?